tests/e2e/cluster_downgrade_test.go

```diff
@@ -77,21 +111,34 @@ func startEtcd(t *testing.T, execPath, dataDirPath string) *e2e.EtcdProcessClust
 			t.Fatalf("error closing etcd processes (%v)", errC)
 		}
 	})
+	return epc
+}
 
+func triggerSnaphsot(t *testing.T, epc *e2e.EtcdProcessCluster) {
 	prefixArgs := []string{e2e.CtlBinPath, "--endpoints", strings.Join(epc.EndpointsV3(), ",")}
-	t.Log("Write keys to ensure wal snapshot is created so cluster version set is snapshotted")
+	var err error
 	e2e.ExecuteWithTimeout(t, 20*time.Second, func() {
 		for i := 0; i < 10; i++ {
```

Comment:
In https://github.com/etcd-io/etcd/pull/13686/files, you are removing snapshotCount from the newCluster method. 

I assume this will cause this method to stop working...